src/base/bittorrent/session.h

```diff
@@ -455,6 +455,11 @@ namespace BitTorrent
         void setBlockPeersOnPrivilegedPorts(bool enabled);
         bool isTrackerFilteringEnabled() const;
         void setTrackerFilteringEnabled(bool enabled);
+        QStringList excludedFileNames() const;
+        void setExcludedFileNames(const QStringList &newList);
+        QVector<QRegExp> excludedFileNamesRegExpList() const;
```

Comment:
Qt doesn't recommends using `QRegExp` anymore, you should use the replacement.

https://doc.qt.io/qt-5/qregexp.html
>Note: In Qt 5, the new [QRegularExpression](https://doc.qt.io/qt-5/qregularexpression.html) class provides a Perl compatible implementation of regular expressions and is recommended in place of QRegExp.

Reply:
>Qt doesn't recommends using `QRegExp` anymore

Qt6 doesn't have it at all so we now use `QRegularExpression`.